src/commons_abstract_fevalues.jl

```diff
@@ -14,10 +14,18 @@ The quadrature rule for the `AbstractFEValues` type.
 * `::QuadratureRule`: the quadrature rule.
 
 """
-getquadrule(fe_cv::FECellValues) = fe_cv.quad_rule
+getquadrule(fe_cv::FECellScalarValues) = fe_cv.quad_rule
```

Comment:
with my suggested hierarchy this could be `getquadrule(fe_cv::AbstractFECellValues) = fe_cv.quad_rule`